picard/album.py

```diff
@@ -552,6 +553,27 @@ def switch_release_version(self, mbid):
             self.tagger.albums[mbid] = self
             self.load(priority=True, refresh=True)
 
+    def update_metadata_images(self):
+        new_images = []
+        orig_images = []
+        for track in self.tracks:
+            for file in list(track.linked_files):
+                for image in file.metadata.images:
```

Comment:
This same for loop has been repeated 4 times, maybe you can make a private function and call it instead?